tests/test_main.py

```diff
@@ -156,10 +171,12 @@ def test_distance_class_iadd_number(kilometers, kilometers2, result):
     [
         (50, 3, 150),
         (30, 7, 210),
-        (45, 5, 225)
+        (45, 5, 225),
+        (1.5, 3.0, 4.5),
+        (21.4, 5.88, 125.832)
```

Comment:
The expected results for floating-point arithmetic operations might not be precise due to floating-point representation errors. Consider using `pytest.approx` for assertions involving floating-point numbers to allow for a small tolerance in the comparison.